test/tasks/before-suite.yaml

```diff
@@ -0,0 +1,107 @@
+# ginkgo-suite is a generic task that executes ginkgo on the given repository and ginkgo suite.
```

Comment:
This looks very similar to the `create-eks.yaml` and `install-karpenter-eks.yaml` files in the examples folder. If we're going to use this, I think we can go ahead and remove those examples, since this looks much better catered to the dev loop.

Reply:
Yes. I expect we'll deprecate all examples at some point. They were helpful for getting me started.